web/admin.py

```
@@ -358,6 +415,33 @@ def rate_limit_status(self, obj):
 
     rate_limit_status.short_description = "Login Status"
 
+    def email_status(self, obj):
```

Ruff (0.8.2)

Missing return type annotation for public function `email_status`

(ANN201)